### PR TITLE
Messaging Message GetMessages

### PR DESCRIPTION
**Pull request by problemv**
For #836

Links to the line numbers/files in the OpenStack source code that support the
code in this PR:

https://github.com/openstack/zaqar/blob/8e0dd9763aba67bb7094cf92312daeba8d087866/zaqar/transport/wsgi/v2_0/messages.py#L254

### CODE REVIEW

**Comment by problemv** [comment]
@jtopjian Sorry about the small hiatus. This is ready for review. :)

Quick note: There is a follow up function just for get which is to get a single message hence this function is called GetMessages.

File: openstack/messaging/v2/messages/requests.go [hunk 1]
```diff
@@ -174,3 +174,37 @@ func PopMessages(client *gophercloud.ServiceClient, queueName string, opts PopMe
...
+
+// GetMessagesOptsBuilder allows extensions to add additional parameters to the
+// GetMessages request.
+type GetMessagesOptsBuilder interface {
+	ToGetMessagesListQuery() (string, error)
+}
+
+// GetMessagesOpts params to be used with GetMessages.
+type GetMessagesOpts struct {
+	Ids []string `q:"ids,omitempty"`
```
Comment (jtopjian) [inline openstack/messaging/v2/messages/requests.go]: Let's use IDs here to match the `Delete` function.

**Comment by problemv** [comment]
@jtopjian No problem my bad! fixed :)

**Comment by jtopjian** [comment]
LGTM - thanks! 

The OpenLab failure is ignorable -- it's a separate issue being worked on.

**Pull request merged**